packages/runtime/container-runtime/src/summary/summarizerNode/summarizerNode.ts

```diff
@@ -406,48 +399,22 @@ export class SummarizerNode implements IRootSummarizerNode {
 					});
 				}
 
-				if (proposalHandle !== undefined) {
-					const maybeSummaryNode = this.pendingSummaries.get(proposalHandle);
-
-					if (maybeSummaryNode !== undefined) {
-						this.refreshLatestSummaryFromPending(
-							proposalHandle,
-							maybeSummaryNode.referenceSequenceNumber,
-						);
-						eventProps.wasSummaryTracked = true;
-						eventProps.latestSummaryUpdated = true;
-						event.end(eventProps);
-						return {
-							latestSummaryUpdated: true,
-							wasSummaryTracked: true,
-							summaryRefSeq,
-						};
-					}
+				let isSummaryTracked = false;
+				let isSummaryNewer = false;
 
-					const props = {
-						summaryRefSeq,
-						pendingSize: this.pendingSummaries.size ?? undefined,
-					};
-					this.logger.sendTelemetryEvent({
-						eventName: "PendingSummaryNotFound",
-						proposalHandle,
-						referenceSequenceNumber: this.referenceSequenceNumber,
-						details: JSON.stringify(props),
-					});
+				if (summaryRefSeq > this.referenceSequenceNumber) {
+					isSummaryNewer = true;
 				}
-
-				// If the summary for which refresh is called is older than the latest tracked summary, ignore it.
-				if (this.referenceSequenceNumber >= summaryRefSeq) {
```

Comment:
Observation:  This case is no longer addressed by this function.  It's up to the caller.  This is probably fine, but I'd want us to think really carefully about the global state here, since `this.referenceSequenceNumber` can change throughout this flow.

I think it's fine because it only changes if we are tracking the incoming summary, in which case before his code would be unreachable anyway.  And if we're not tracking, then it doesn't change, so it's fine for the caller to check.

Sound correct?  Hopefully test coverage helps here too.

Reply:
Weird that this was `>=`.  Not that they should be able to be equal but not the same (in which case it would be tracked). But you never know what could happen with Summaries!

Reply:
This check is still there:
```
if (summaryRefSeq > this.referenceSequenceNumber) {
  isSummaryNewer = true;
}
```
`isSummaryNewer` is not returned because the caller has the information to gather this information. I could change it to return this value but I don't think its needed (infact I had it this way in my local repo and changed it).
IMO, the caller should decide what to do based on the return value `isSummaryTracked`. If the summary is tracked, the latest summary information (referenceSequenceNumber) will be updated and if not, it won't be.

Reply:
Added `isSummaryNewer` to the return of this function and check that at the call site.

Reply:
I'm curious Why you brought it back? I thought it was fine after I thought out loud about it. It was just a little hard to trace through the diff.

Reply:
I was on the fence about keeping it vs removing it from the point of view of not making significant changes in one go. With you comment, it seemed like it added some confusion so I added it back. We can remove it later if needed.